.github/workflows/test.yml

```diff
@@ -0,0 +1,77 @@
+name: Targeted Tests
```

Comment:
This should have a more clear file name than 'test.yml' - most if not all workflows in this directory are testing-related